src/plugins/new-ui/public/webcomponents/depth.html

```diff
@@ -30,27 +30,38 @@
           1px  1px 0 #fff;
       }
       #depthTarget {
-        font-size: 1vw;
+        font-size: 0.65vw;
         font-weight: bold;
+        text-align: left;
+        margin-top: 0.5vw;
         /*padding-top: 2px;*/
       }
       #depthTarget.hidden {
         opacity: 0;
       }
       #depth-footer  {
         width: 100%;
-        margin-top: -20px;
+        margin-top: 1.0vw;
+
+        font-size: 1vw;
       }
       #water-type {
-        width: 35%;
-        float: left;
+        font-weight: bold;
       }
       #depth-unit {
-        width: 35%;
-        text-align: right;
+        width: 15%;
+        text-align: left;
         float: right;
+        margin-top: 0.45vw;
+
+        font-size: 1.0vw;
```

Comment:
Assuming we put the units back next to the depth reading, the only css setting that changes anything is font-size, all the rest can be removed.